s3_to_knack.py

```diff
@@ -173,13 +173,16 @@ def coalesce_records(records_current, coalesce_fields, current_pk, separator = "
                 coal_record[field] = separator.join([coal_val, current_val])
             elif current_val:
                 coal_record[field] = current_val
-    return [val for key, val in index.items()]
+    return list(index.values())
```

Comment:
this caught my eye and i couldn't resist making it more idiomatic — it's functionally equivalent without the unused `key` var.